include/villas/node.hpp

```diff
@@ -7,6 +7,7 @@
 
 #pragma once
 
+#include <filesystem>
```

Comment:
Please note that we currently do not support the STL filesystem library due to compiler restrictions. GCC 8.3 unfortunately sometimes ships without it :(

For DPsim, I started to check this with CMake and add a standalone implementation which is compatible. But for now it would be nice to try get around without it.

Reply:
I understand. I can change all uses of std::filesystem to std::string. However, superNode and Config already use std::filesystem for a few months and I have heard no issues. GCC 8.3 is really old - I wouldn't feel too bad to drop support for this. If someone complains it's pretty easy to replace std::filesystem with std::string.

Reply:
I already patched-out the `std::filesystem` usage in a non-public branch for about the same time :D I know GCC 8.3 is pretty old :( But its unfortunately the version which is shipped with our OPAL-RT systems. I expect they we upgrade to a newer version with the next release of the OPAL-RTLinux OS.

Can we stick with `std::string` for now? I'm also volunteering to introduce `std::filesystem` once we support it 😄 I am fully on your side, that it makes the code nicer..

Reply:
@n-eiling I've cherry-picked the removal of `std::filesystem` in https://github.com/VILLASframework/node/pull/730

Reply:
ok thanks. will change it on this branch.

Reply:
ah you also changed it on this branch. even better :)

Reply:
what about in VILLASfpga? We will never use that on OPAL targets, right?